src/training/niah_probe.py

```
@@ -1,95 +1,103 @@
 
     def __init__(
         self,
         dim: int,
         probe_frequency: int = 1000,
-        haystack_size: int = 100,
-        accuracy_threshold: float = 0.8,
+        haystack_size: int = 4,
+        accuracy_threshold: float = 0.1,
+        seq_len: int = 128,
+        # Legacy params accepted but ignored for backward compat
+        ttl_steps: int = 10,
```

Ruff (0.14.14)

Unused method argument: `ttl_steps`

(ARG002)